README.md

```diff
@@ -12,16 +12,36 @@ oc process -f https://raw.githubusercontent.com/jaegertracing/jaeger-openshift/m
 Once everything is ready, `oc status` tells you where to find Jaeger URL.
 
 ## Production setup
-This template deploys the Collector, Query Service (with UI) and Cassandra storage (StatefulSet) as separate individually scalable services.
 
-Install everything in `jaeger-infra` namespace:
+This setup requires a backing storage (Cassandra or Elasticsearch) to exist and be in stable mode before it can be deployed.
+For the sake of completeness, one template for each backing storage is provided but none is supported at this moment. The storage
+templates should not be seen as "production ready". Refer to the storage's own documentation and OpenShift/Kubernetes examples
+on how to appropriately plan and deploy them in your environment.
+
+The production template deploys the Collector and Query Service (with UI) as separate individually scalable services.
+
+### Example 1 - Install a stable version of Jaeger, with Cassandra as backing storage
```

Comment:
Why not just  `Install a stable version of Jaeger, with Cassandra as backing storage`?

Reply:
Not quite sure I get what the alternative would be. Are you suggesting to have only this example and remove the other one? Or are you suggesting to remove "Example 1" from the title?

Reply:
remove example from the title